src/getProfile.js

```diff
@@ -0,0 +1,40 @@
+import { fetchProfiles } from './utils';
+
+const HOUR = 60 * 60
+
+export const getProfile = async (request, env, ctx) => {
+  const body = await request.json();
+  const relayUrls = body.relays.map(url => (new URL(url)).origin);
+  const pubkey = body.pubkey;
+
+  if (!relayUrls || !pubkey) {
+    return new Response('INVALID_REQUEST', { status: 422 });
```

Comment:
I think in index.js#20 you expect a JSON return value and not a response object.

even though it might be good to expect a response object and just return that instead of creating the response object in index.js

https://github.com/getAlby/http-nostr-publisher/pull/1/files?diff=split&w=0#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R20-R24

Reply:
Sure, changing!